lisa/features/disks.py

```diff
@@ -54,6 +54,9 @@ def get_all_disks(self) -> List[str]:
     def get_hardware_disk_controller_type(self) -> schema.DiskControllerType:
         raise NotImplementedError
 
+    def get_ephemeral_disk_placement_type(self) -> schema.EphemeralDiskPlacementType:
```

Comment:
It looks an Azure only feature, please add it in the inherited Azure disk feature.